shared_requirements.txt

```diff
@@ -303,3 +303,24 @@ opentelemetry-sdk<2.0.0,>=1.5.0,!=1.10a0
 #override azure-mgmt-sqlvirtualmachine azure-mgmt-core>=1.3.0,<2.0.0
 #override azure-mgmt-rdbms msrest>=0.6.21
 #override azure-mgmt-rdbms azure-mgmt-core>=1.3.0,<2.0.0
```

Comment:
We could remove a lot of these `azure-mgmt-core` overrides with this change; up to you!

Reply:
good point, let me do that!

Reply:
Analyze dependencies step is passing CI, merging this